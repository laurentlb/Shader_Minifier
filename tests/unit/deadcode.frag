int f(int x) {
#if FOO
    return x;
#else
    return x + 2;
#endif
}

void main() {
    int a = 2;
    f(a);
    return;
    f(a + 1);
}
