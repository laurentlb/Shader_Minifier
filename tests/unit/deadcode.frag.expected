int f(int x)
{
  
#if FOO

  return x;
  
#else

  return x+2;
  
#endif

}
void main()
{
  f(2);
  return;
}
